crypto/multisig.go

```diff
@@ -249,37 +261,18 @@ func MultisigBatchPrep(msg Hashable, addr Digest, sig MultisigSig, batchVerifier
 		return errInvalidNumberOfSignature
 	}
 
-	// check that we don't have too few multisig subsigs
-	if len(sig.Subsigs) < int(sig.Threshold) {
-		return errInvalidNumberOfSignature
-	}
-
 	// checks the number of non-blank signatures is no less than threshold
-	var counter uint8
-	for _, subsigi := range sig.Subsigs {
-		if (subsigi.Sig != Signature{}) {
-			counter++
-		}
-	}
-	if counter < sig.Threshold {
+	if sig.Signatures() < int(sig.Threshold) {
```

Comment:
this appears to be a protocol change - old and new version process blank multisig txns differently

Reply:
What is the difference? `sig.Signatures()` does the same thing as the loop, so this line do the same thing as it was before. If there are no signatures (and sig.Threshold is > 0) then `errInvalidNumberOfSignature` is returned.

Reply:
I was thought about blank signatures in this loop above:
```
	if len(sig.Subsigs) < int(sig.Threshold) {
		return errInvalidNumberOfSignature
	}
```
Now I see, this is only a preliminary check and the new and old code are functionally equivalent, sorry.

Reply:
No problem, I always appreciate a double-check.